src/core/store/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useReducer, useContext } from 'react';
```

Comment:
This is really awesome that you've created such a well-thought structure with ContextAPI!
It will play a really important role with the introduction of Redux.